src/types/widget.ts

```diff
@@ -473,7 +172,9 @@ export type Widget<
    */
   getWidgetSearchParameters?(
     state: SearchParameters,
-    widgetSearchParametersOptions: WidgetSearchParametersOptions
+    widgetSearchParametersOptions: {
```

Comment:
Any reason you removed `WidgetSearchParametersOptions` type and access it via `Parameters` later?

Reply:
the type later (PR to come) depends on the generic passed to Widget, so the WidgetSearchParametersOptions would need the generic too. I might revisit this part once its migrated fully!